enlace/include/settings.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/*
+ *   TTM is a web application to manage data collected by community organizations.
+ *   Copyright (C) 2014, 2015  Local Initiatives Support Corporation (lisc.org)
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Affero General Public License as published by
+ *   the Free Software Foundation, either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU Affero General Public License for more details.
+ *
+ *   You should have received a copy of the GNU Affero General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+?>
+<?php
+
+if (!function_exists("get_setting")) {
+    /*
+     * For the following, we exception out with improper settings because we shouldn't be adding code that
+     * uses settings if the settings aren't specifically in the database, and we should use the global names
+     * rather than passing in ad hoc strings.
+     */
+
+    $NumDaysHiddenSetting = "num_days_hidden";
```

Comment:
IOW, this assignment creates a global variable `$NumDaysHiddenSetting`, even though it takes place within the scope of the `if` block's curly braces?  (I'm not super up on PHP scoping rules, so just checking.)